### PR TITLE
View card details

### PR DESCRIPTION
**Pull request by zpawn**
Link to Shortcut: https://app.shortcut.com/deskpro/story/74930/trello-view-card-details

### CODE REVIEW

File: src/components/ViewCard/ViewCard.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+import { Members } from "../common/Cards";
+
+const EmptyLabel = styled.span`
+    display: inline-block;
+    min-width: 20px;
+    content: " ";
+`;
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const ViewCard: FC<any> = ({
```
Comment (AshleyDawson) [inline src/components/ViewCard/ViewCard.tsx]: ViewCard props need to be defined, e.g. 

```ts
interface ViewCardProps {
  // ...
}
```

**Review by AshleyDawson — CHANGES_REQUESTED** [review]
TS type changes required

File: src/components/ViewCard/ViewCard.tsx (hunk 1 above)
Comment (zpawn) [reply src/components/ViewCard/ViewCard.tsx]: Ok, I'll add

Comment (zpawn) [reply src/components/ViewCard/ViewCard.tsx]: done

**Review by AshleyDawson — APPROVED** [review]

**Pull request merged**